weed/server/filer_server_handlers_write_upload.go

```diff
@@ -197,6 +198,15 @@ func (fs *FilerServer) dataToChunk(fileName, contentType string, data []byte, ch
 	})
 	if err != nil {
 		glog.Errorf("upload error: %v", err)
+		if strings.Contains(err.Error(), "failed to write to replicas for volume") {
```

Comment:
there could be multiple retries. Need to put this within the util.Retry section.